Tella/Scenes/HomeView/Views/FileListView/Add files/PhotoVideoViewModel.swift

```diff
@@ -44,34 +50,163 @@ class PhotoVideoViewModel : ObservableObject {
             }
         }
     }
-    
+    /// This function imports the video file from the user selected video with the without metadata attached to the file
+    /// - Parameters:
+    ///   - files: Array of the URL of the videos
+    ///   - type: Type of file
+    func addVideoWithoutExif(files: [URL], type: FileType) {
+        files.forEach { file in
+            let tmpFileURL = URL(fileURLWithPath:NSTemporaryDirectory()).appendingPathComponent("\(Int(Date().timeIntervalSince1970))").appendingPathExtension(file.lastPathComponent)
+            Task {
+                do {
+                    guard let url = await self.exportFile(url: file, destinationURL: tmpFileURL) else { return }
+                    let vaultFile = try await self.mainAppModel.add(files: [url],
+                                                                    to: self.mainAppModel.vaultManager.root,
+                                                                    type: type,
+                                                                    folderPathArray: self.folderPathArray)
+
+                    if self.mainAppModel.importOption == .deleteOriginal {
+                        self.removeFiles(files: files)
+                    }
+                    DispatchQueue.main.async {
+                        self.resultFile?.wrappedValue = vaultFile
+                    }
+                }
+                catch {
+
+                }
+            }
+        }
+    }
+
+    /// This function imports the image file from the user selected image with the metadata attached to the file
+    /// - Parameters:
+    ///   - image: The UIImage of the image file selected
+    ///   - type: Type of the file that is selected
+    ///   - pathExtension: File extension
+    ///   - originalUrl: The original URL of the image file
+    ///   - acturalURL:  The actual URL of the image file
+    func addWithExif(image: UIImage , type: FileType, pathExtension:String?, originalUrl: URL?, actualURL: URL?) {
```

Comment:
I think we can use the same logic in video and image instead to use UImage for Image and URL for Video,
What do you think? I feel that there is unecessary code to convert UImage and we have directly the image URL.